releasenotes/notes/pyramid-config-7db0cb8f089c9417.yaml

```diff
@@ -0,0 +1,4 @@
+---
+other:
+  - |
+    Added distributed tracing setting to integration config for pyramid.
```

Comment:
Probably good to be specific here again. 

"Added `config.pyramid.distributed_tracing` setting"